server/src/lib/model/db.ts

```diff
@@ -413,6 +413,13 @@ export default class DB {
     id: string,
     auth_token?: string
   ): Promise<boolean> {
+    const guidRegex = /^[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/;
+    const authRegex = /^[0-9a-f]{40}$/;
+
+    if (!guidRegex.test(id) || (auth_token && !authRegex.test(auth_token))) {
```

Comment:
Is it every likely for the id/token to not be all lowercase? It might be safer to just do a quick transform to lowercase, just in case.

Reply:
>     * How many of those nonstandard accounts have emails attached (I don't think we allow login without email anymore)

…exactly ZERO accounts.

>     * How many of them have any associated entries in user_client_activities (which will tell us when they were last on site)

139 accounts, responsible for 5781 activities. Some as recent as a few days ago.

>     * How many of them have any associated clips or votes (since deleting user accounts will cascade everywhere else)

13 have associated clips (12 of which are from telegram), 1 has associated votes, none of this activity occurred within the past year.

Reply:
Discussed offline and updated `authRegex` 👍